pkg/skaffold/watch/watch_test.go

```diff
@@ -108,8 +108,9 @@ func newCallback() *callback {
 	}
 }
 
-func (c *callback) call() {
+func (c *callback) call(e Events) error {
```

Comment:
what's this change for?

Reply:
callback.call() has to implement ChangeFn which is func(Events) error